### ISSUE TITLE
Unable to change a value

### ISSUE THREAD

**Issue by Lupias** [cheat-engine/cheat-engine#2086]
I was trying to change a value in this game and whenever i click scan i just hear a ding and nothing happens, i have tried changing what byte it uses while scanning but nothing helps

**Comment by fade2gray** [comment]
Have you tried the tutorials in the help menu to get a better understanding of what you want to achieve?

**Comment by Lupias** [comment]
> Have you tried the tutorials in the help menu to get a better understanding of what you want to achieve?

Yes i"ve tried the tutorial and worked just fine but doesn't seem to work anywhere else.

**Comment by danisbored** [comment]
> > Have you tried the tutorials in the help menu to get a better understanding of what you want to achieve?
> 
> Yes i"ve tried the tutorial and worked just fine but doesn't seem to work anywhere else.

When scanning for values a lot of values will show up (in most of the games), maybe you're modifying the wrong one? Try watching tutorials to get a better undestanding of how to CE and manipulation works to archive your goal  (or try the tutorial in CE)
